content/300-guides/050-database/850-using-prisma-with-planetscale.mdx

```diff
@@ -40,7 +40,8 @@ PlanetScale's branching model and design for scalability means that there are al
 
 - **Branching and deploy requests.** PlanetScale provides two types of database branches: _development branches_, which allow you to test out schema changes, and _production branches_, which are protected from direct schema changes. Instead, changes must be first created on a development branch and then deployed to production using a deploy request. Production branches are highly available and include automated daily backups. To learn more, see [How to use branches and deploy requests](#how-to-use-branches-and-deploy-requests).
 
-- **Referential actions and integrity.** To support scaling across multiple database servers, PlanetScale [does not allow the use of foreign key constraints](https://docs.planetscale.com/learn/operating-without-foreign-key-constraints), which are normally used in relational databases to enforce relationships between data in different tables. To maintain these relationships in your data while using PlanetScale with Prisma and allow the use of [referential actions](/concepts/components/prisma-schema/relations/referential-actions), you will need to use Prisma's ability to [emulate referential integrity in the Prisma client](/concepts/components/prisma-schema/relations/referential-integrity#handling-the-referential-integrity-in-prisma). For more information, see [How to enable emulation of referential integrity](#how-to-enable-emulation-of-referential-integrity).
+- **Referential actions and integrity.** To support scaling across multiple database servers, PlanetScale [_does not allow the use of foreign key constraints_](https://docs.planetscale.com/learn/operating-without-foreign-key-constraints), which are normally used in relational databases to enforce relationships between data in different tables, and asks users to handle this manually in their applications.   
```

Comment:
```suggestion
- **Referential actions and integrity.** To support scaling across multiple database servers, PlanetScale [does not allow the use of foreign key constraints](https://docs.planetscale.com/learn/operating-without-foreign-key-constraints), which are normally used in relational databases to enforce relationships between data in different tables, and asks users to handle this manually in their applications.   
```

Happy with the rewording but the italics in links look very odd to me, don't think we are using that elsewhere...

Reply:
I wanted to highlight the fact what is actually the difference. I agree that link plus italics is not the most common, but 🤷 

You decide if you want to undo that change - happy either way.